public/index.css

```diff
@@ -181,14 +181,11 @@ strong {
 
 /* Supporting Content */
 code {
-	font-family: var(--font-mono);
-	font-size: 0.85em;
-}
-
-code:not([class*='language']) {
```

Comment:
Are you sure about removing this? Just confirming since this previously styled inline code but not larger code blocks.